### ISSUE TITLE
Question regarding the process of reading out data

### ISSUE THREAD

**Issue by m1iles** [RayenHML/CAN-Steering-Angle-Reader-VW-Passat#1]
Hi,
I have a project which requires me to read out the current mileage value and fuel tank value. I found those values in the same document you used.
Now comes my question. I am quiet new to OBD2 and CAN messages and wanted to ask if the data you use is requested or you just listen to the "broadcast" and then extract the data once it is broadcasted. I hope this makes sense and you know what I mean.
Thanks in advance.
Best regards
Kilian

**Comment by m1iles** [comment]
Also one more thing: what year was your passat assembled? The one I am trying to read out that data is from 2018.

**Comment by RayenHML** [comment]
Hi @m1iles, 

I'm glad to answer your questions:
1) Yes, I am actually listening to what is being broadcast. With this method, you don't have to send a request and then receive the desired signal back.
2) Our VW Passat is also from the year 2018.

**Comment by m1iles** [comment]
Hi @RayenHML,
thanks for the response. Yesterday I tried to listen to the canbus and when I used (https://github.com/linux-can/can-utils) the candump command but I only saw the following broadcasted: Current speed, current engine load, intake temp, system status and current rpm. I got that data when listening to "can0". I hope this makes sense.

I think there should be a different way to connect to a different canbus to get more data right? What was your way of connecting to the can?

Thanks in advance, hopefully I am not annoying :)
